src/service.ts

```diff
@@ -206,14 +206,15 @@ export const snowballRService: ISnowballR = {
         const { oldPassword, newPassword } = call.request;
         const user = getAuthenticated(call.metadata)!;
 
-        if (user.password == oldPassword) {
-            USERS.get(user.email)!.password = newPassword;
-            callback(null, {});
-        } else {
+        if (user.password != oldPassword) {
```

Comment:
Is there a reason why we don't use the identity check?
```suggestion
        if (user.password !== oldPassword) {
```
There are many other cases in this file where this applies too. Just search for `==` and `!=`.

Reply:
I noticed this throughout the repo. Was there a reason for this?